src/test/pretty/stmt_expr_attributes.rs

```diff
@@ -259,8 +259,6 @@ fn _12() {
     }
 }
 
-/////////////////
```

Comment:
The changes in `src/test/` are all good; can you `cherry-pick` those out to a separate PR for fast-tracking?

Reply:
I've done this in #63411